tasks/pkg-debian.yml

```diff
@@ -2,7 +2,11 @@
 - apt: name=apt-transport-https state=latest
 
 - apt_key: id=C7A7DA52 keyserver=hkp://keyserver.ubuntu.com:80 state=present
+  when: apt_key_url is not defined
 
-- apt_repository: repo='deb http://apt.datadoghq.com/ stable main' state=present update_cache=yes
+- apt_key: url={{ datadog_apt_key_url }} state=present
```

Comment:
I don't think the `id` option should be removed here: the id of the key should be identical even when you pull the key from a mirror. Do you have a use case where that wouldn't be true?